ui-lib/src/main/res/ui/export_data/values/strings.xml

```diff
@@ -0,0 +1,16 @@
+<resources>
+    <string name="export_data_header">Consent for Exporting Private Data</string>
+    <string name="export_data_text_1">By clicking \"I Agree\" below, you give your consent to export Zashi’s private
+        data which includes the entire history of the wallet, all private information, memos, amounts and recipient
+        addresses, even for your shielded activity.*\n\nThis private data also gives the ability to see certain future
+        actions you take with Zashi.\n\nSharing this private data is irrevocable — once you have shared this private
+        data with someone, there is no way to revoke their access.</string>
+    <string name="export_data_text_2">*Note that this private data does not give them the ability to spend your
+        funds, only the ability to see what you do with your funds.</string>
+    <string name="export_data_confirm">Export private data</string>
+    <string name="export_data_agree">I agree</string>
+    <string name="export_data_back">Back</string>
```

Comment:
I have reviewed this and it seems like reasonable wording for the consent request.